.github/ISSUE_TEMPLATE/new-feature.yml

```diff
@@ -1,32 +1,97 @@
 name: "💡 New Feature Request"
 description: Suggest new functionality and features for the product.
 labels: ["Type/NewFeature"]
-body:
+body:        
   - type: textarea
-    id: problem
+    id: problem_description
     attributes:
-      label: Problem
-      description: What is the problem this feature will solve?
+      label: What problem are we trying to solve?
+      description: Clear description of the problem. Focus on the problem, not the solution.
+      placeholder: Describe the specific problem or challenge that needs to be addressed...
     validations:
       required: true
+      
   - type: textarea
-    id: solution
+    id: affected_users
     attributes:
-      label: Proposed Solution
-      description: Describe the solution you'd like to have.
+      label: Who is experiencing this problem?
+      description: List affected user types
+      placeholder: e.g., Developers, System administrators, End users, etc.
     validations:
       required: true
+      
   - type: textarea
-    id: alternatives
+    id: impact
     attributes:
-      label: Alternatives
-      description: Describe any alternatives have you considered
+      label: What is the impact?
+      description: Quantify the impact - time wasted, deals blocked, users affected
+      placeholder: e.g., Users spend 30 minutes daily on manual workarounds, blocking 50% of new integrations...
+    validations:
+      required: true
+        
+  - type: textarea
+    id: justification
+    attributes:
+      label: Why Solve This Now? (Strategic alignment / Business justification)
+      description: Describe the strategy, customer requests, or competitive pressure motivating this request.
+      placeholder: e.g., Aligns with our identity management strategy, competitive gaps...
+    validations:
+      required: true
+        
+  - type: textarea
+    id: persona
+    attributes:
+      label: Who Is This For?
+      description: Name, role, goal, pain, and success criteria
+      placeholder: |
+        **Name and role**: e.g., Backend Developer
+        **Goal**: What they want to achieve
+        **Pain**: Current problem they face
+        **Success criteria**: How they'll measure success
+    validations:
+      required: true
```

Comment:
Whats the difference between this and the section "Who is experiencing this problem?"?

Better to have reduce the required fields for the issue creation, without having overlapping questions.